cmd/kubectl-gadget/profile/cpu.go

```diff
@@ -15,9 +15,11 @@
 package profile
```

Comment:
BTW, could you please update the guide with the new CLI output and use the `--timeout` flag? Thanks!